### ISSUE TITLE
PEAR download restriction

### ISSUE THREAD

**Issue by FabWei** [hsgweon/pipits#5]
Hello,

PEAR cannot be downloaded with wget at the moment. They also have a new homepage where you are supposed to register and get your academic-usage link to PEAR tar via mail later. I tried this two times, it worked the second time. The path for creating a link in pipits/bin is also slightly different now.
New PEAR page: https://www.h-its.org/downloads/pear-academic/

Cheers,
Fabian

**Comment by hsgweon** [comment]
Thank you Fabian,

I am not a big fan of restricted licences, so I have now decided to not use PEAR as part of PIPITS. Incidentally, I found VSEARCH's joiner algorithm produces very similar to PEAR (if not exactly the same in many cases). In fact, VSEARCH's authors do openly say that it's algorithm is very similar to PEAR. I am currently doing some rigorous comparisons to make sure they do indeed give the same/similar results (so far the two methods are very similar with a bit of parameter tweaking) and once satisfied, I will then make it as a default joiner as soon as I can (with an option to use PEAR).

Thanks,
Soon
